### PR TITLE
Support parsing a single resource.

### PR DESCRIPTION
**Pull request by grahamlyus**
Thanks for the great library. This change supports parsing a single resource object as well as arrays of resource objects already implemented.

### CODE REVIEW

File: tests/tests.js [hunk 1]
```diff
@@ -260,6 +260,33 @@ describe(
                 );
 
                 describe(
+                    'it should parse a single resouce',
```
Comment (pjfreeze) [inline tests/tests.js]: "resouce" -> "resource"

**Comment by pjfreeze** [comment]
If you fix the typo in the test, I'll go ahead and merge it in and create a new release/tag!

**Comment by grahamlyus** [comment]
Done. Nice spot!

**Comment by pjfreeze** [comment]
Thanks for adding this in!

**Pull request merged**